source/plugins/ruby/ApplicationInsightsUtility.rb

```diff
@@ -133,16 +138,23 @@ def initializeUtility()
     end
 
     def getContainerRuntimeInfo()
-      containerRuntime = ENV[@@EnvContainerRuntime]
-      if !containerRuntime.nil? && !containerRuntime.empty?
-        # DockerVersion field holds either containerRuntime for non-docker or Dockerversion if its docker
-        @@CustomProperties["DockerVersion"] = containerRuntime
-        if containerRuntime.casecmp("docker") == 0
-          dockerInfo = DockerApiClient.dockerInfo
-          if (!dockerInfo.nil? && !dockerInfo.empty?)
-            @@CustomProperties["DockerVersion"] = dockerInfo["Version"]
+      begin
+        # Not doing this for windows since docker is being deprecated soon and we dont want to bring in the socket dependency.
+        if !@@isWindows.nil? && @@isWindows == false
+          containerRuntime = ENV[@@EnvContainerRuntime]
```

Comment:
CONTAINER_RUNTIME available on windows. only thing we need to avoid talking docker sock as its specific to linux and it will be deprecated soon with containerd.

Reply:
I will fix this.